cmake/singularity-eos/spiner.cmake

```diff
@@ -0,0 +1,46 @@
+macro(_singularity_set_spiner_opt singularity_opt spiner_opt)
+  if(${singularity_opt})
+    set(${spiner_opt}
+        ON
+        CACHE BOOL "" FORCE)
+  endif()
+endmacro()
+
+macro(singularity_import_spiner)
+  if(NOT TARGET spiner::spiner)
+    _singularity_set_spiner_opt(SINGULARITY_USE_SPINER_WITH_HDF5 SPINER_USE_HDF)
+    # TODO I dunno if these are used
+    _singularity_set_spiner_opt(SINGULARITY_USE_KOKKOS SPINER_USE_KOKKOS)
+    _singularity_set_spiner_opt(SINGULARITY_USE_CUDA SPINER_USE_CUDA)
```

Comment:
They aren't used anymore. It's all passed through ports of call.

Reply:
Commented out for now